test/integration/master/kube_apiserver_test.go

```diff
@@ -85,6 +86,55 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func endpointReturnsStatusOK(client *kubernetes.Clientset, path string) bool {
+	var status int
+	res := client.CoreV1().RESTClient().Get().AbsPath(path).Do()
+	status = 0
```

Comment:
is this necessary?

Reply:
Is there another way to derive the status code?

Reply:
The way is fine. Was just questioning the assignment of zero. It's already done by the declaration.

Reply:
oh duh. yeah sure, i can remove it.